tests/storage-compat/gen_storage_compat_data.sh

```diff
@@ -39,7 +39,7 @@ done
 # run python script to populate db
 IMAGE_NAME=$(docker buildx build --load -q "${SCRIPT_DIR}/populate_db")
 docker run --rm \
-            -e QDRANT_HOST=host.docker.internal:6333 \
```

Comment:
I could not get the current version work on my setup without this cange

Reply:
That's weird, the main reason I changed it before was to make it work in both linux and osx, since `network=host` does not work in osx. I wonder if there is an alternative

Reply:
For info, here are the logs:

The resolved address was `http://host.docker.internal:6333/collections/test_collection`

```
++ docker buildx build --load -q /home/agourlay/Workspace/qdrant/tests/storage-compat/populate_db
+ IMAGE_NAME=sha256:da1874823964b2ec5d4a9855be1d50974fe86bdd5ab3274fdc70e750bdc72747
+ docker run --rm -e QDRANT_HOST=host.docker.internal:6333 --network=host --add-host host.docker.internal:host-gateway sha256:da1874823964b2ec5d4a9855be1d50974fe86bdd5ab3274fdc70e750bdc72747 sh -c 'python populate_db.py'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/urllib3/connection.py", line 203, in _new_conn
    sock = connection.create_connection(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/urllib3/util/connection.py", line 85, in create_connection
    raise err
  File "/usr/local/lib/python3.11/site-packages/urllib3/util/connection.py", line 73, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
```

Reply:
I propose to revert this change because you have already regenerated storage and docker does not support `--network="host"` for mac os:
https://docs.docker.com/network/drivers/host/

Reply:
Alright, I will revert this change before merging :+1: